portfolio/src/main/webapp/projects.html

```diff
@@ -121,7 +118,12 @@ <h4>Computer Science Fundamentals Website</h4>
                 </p>
             </li>
         </ul>
-      </p>
+    </p>

```

Comment:
I didn't find a matching opening tag for </p>. Shall we remove this?

Reply:
Removed extra tag.